app/pubspec.yaml

```diff
@@ -133,6 +133,7 @@ dependencies:
   shake_detector:
     path: ../packages/shake_detector
   open_filex: ^4.5.0
+  phosphor_flutter: ^2.1.0
```

Comment:
In recent past, I have explore this package and check with Acter app. Here is the my findings.

Advantages:
- Phosphor have better look and feel compare to Atlas
- Phosphor have more number of icons compare to Atlas

Disadvantages:
- Phosphor have less variety for single icon category

Reply:
>     Phosphor have less variety for single icon category

I don't understand what that means. Can you elaborate?

Reply:
Lets example of icons with `Email` category. Atlas have more variety in single category.

![Icons](https://github.com/user-attachments/assets/a4802eaa-ee7f-4a70-bc60-e570b68799c2)

Reply:
ah. gotcha ... not sure that matter too much though...

Reply:
https://github.com/user-attachments/assets/28e670b7-a1aa-46a3-a8e2-e9c4c7ddaeee